proto/autokitteh/projects/v1/svc.proto

```diff
@@ -113,6 +113,30 @@ message ExportResponse {
   bytes zip_archive = 1 [(buf.validate.field).bytes.min_len = 10];
 }
 
+message LintRequest {
+  string project_id = 1 [(buf.validate.field).string.min_len = 1];
+  map<string, bytes> resources = 2;
```

Comment:
if this is on the project, why do you need resources and manifest file? do it directly on the saved resources?